ffc-pay-alerts/notify.js

```
@@ -0,0 +1,23 @@
+const {
+  notifyApiKey,
+  notifyEmailTemplateId,
+  notifyEmailAddress
+} = require('./config')
+
+const NotifyClient = require('notifications-node-client').NotifyClient
+const notifyClient = new NotifyClient(notifyApiKey)
+
+const sendEmail = async (context, message, emailAddress = notifyEmailAddress, reference = '') => {
+  try {
+    await notifyClient.sendEmail(notifyEmailTemplateId, emailAddress, {
+      personalisation: message,
+      reference
+    })
+    context.log(`Email sent to ${emailAddress} containing `, message)
+  } catch (err) {
+    const errMessage = `Oh dear. ${err}`
```

What error message should appear here?

---

I would just throw the error so the function fails and the message is not completed from service bus.